packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,737 @@
+| name        | String                                                                                | Human-readable name for the Rule, configurable during creation or updating.                                        |
+| id          | String                                                                                | Unique identifier for this Rule (read-only).                                                                       |
+| type        | String                                                                                | Type of the policy. For Identifier Match Rule Objects, this needs to be `Okta:IdentifierMatch`.                    |
+| priority    | Integer                                                                               | Used to determine which Rules take precedence.                                                                     |
+| conditions  | Array                                                                                 | Identifier and App conditions are supported for this Rule type.                                                    |
+| action      | String                                                                                | Either `ALLOW` or `DENY`. Controls whether the user is allowed to proceed.                                         |
+| requirement | [Identifier Match Rule Requirement Object](#identifier-match-rule-requirement-object) | Specifies the user profile attributes to match against, as well as the action to take in case of multiple matches. |
+| status      | String                                                                                | `ACTIVE`  or  `INACTIVE`.                                                                                          |
+| default     | Boolean                                                                               | `true` for the first instance of this Rule, which gets created by default.                                         |
+
+#### Identifier Match Rule Requirement Object
+
+| Property              | Type   | Description                                                                                                                                 |
+|-----------------------|--------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| identifyingAttributes | Array  | User profile attributes to match. For each user profile attribute, you need to specify the `userType` ID, as well as the  `attribute` name. |
+| onConflictingUser     | String | Whether to proceed if more than one match is found. Can be `ALLOW` or `DENY`.|                                                              |                                                                       |
```

Comment:
shall we briefly mention how the pipeline will proceed when more than one match is found (iirc, pick the first one) ?

Reply:
There is a nested `action` attribute inside the  `onConflictingUser` object, and it currently only support `DENY`

Reply:
Changed to:
```
| `onConflictingUser.action` | String | Whether to proceed if more than one match is found. Currently only `DENY` is supported.|
``